otter/convergence/composition.py

```diff
@@ -22,12 +22,12 @@ def tenant_is_enabled(tenant_id, get_config_value):
         "convergence-tenants" portion of the configuration file.
     :param callable get_config_value: config key -> config value.
     """
-    enabled_tenant_ids = get_config_value("convergence-tenants")
```

Comment:
I recommend making this an additive change so we can switch from a whitelist to a blacklist with only a config change.

Here's a suggestion for how: `convergence-enabled` could be a new config key, the value would either be `"blacklist"` or `"whitelist"`, and then there would also be `convergence-tenants` and `non-convergence-tenants` keys.

```python
if convergence_enabled == 'blacklist' and tenant_id not in non_convergence_tenants:
    return True
elif convergence_enabled == 'whitelist' and tenant_id in convergence_tenants:
    return True
else:
    return False
```